### PR TITLE
Report kubernetes system metadata

### LINKED ISSUE #181: Report kubernetes system metadata
per elastic/apm#21

### PR DESCRIPTION
**Pull request by vhatsura**
closes #181 

TBD:
- [x] tests
- [x] documentation

### CODE REVIEW

**Comment by vhatsura** [comment]
@bmorelli25, I don't sure do we need to add it into documentation? I found nothing in other agents related to it.

**Comment by bmorelli25** [comment]
We document Kubernetes metadata centrally, in APM Server: https://www.elastic.co/guide/en/apm/server/master/metadata-api.html. I don't think anything is needed in this PR 👍

**Comment by vhatsura** [comment]
@gregkalapos, I'll very much appreciate it if you can review the PR.

**Comment by gregkalapos** [comment]
Hey @vhatsura - sorry for the delay, currently we are fighting with some CI issues; hopefully we can get back to normal soon. Once master is green again I'll make sure PRs gets reviewed and merged.

**Comment by vhatsura** [comment]
@gregkalapos, just a kindly reminder. Would be nice to have such feature in the next release

**Comment by ejsmith** [comment]
Bump

File: src/Elastic.Apm/Helpers/SystemInfoHelper.cs [hunk 1]
```diff
@@ -72,8 +71,16 @@ private Container ParseContainerId(string line)
...
-		internal Api.System ParseSystemInfo() =>
-			new Api.System { Container = ParseContainerInfo(), DetectedHostName = GetHostName() };
+		internal Api.System ParseSystemInfo()
+		{
+			var containerInfo = ParseContainerInfo();
+			var hostName = GetHostName();
+
+			return new Api.System
+			{
+				Container = containerInfo, DetectedHostName = hostName, Kubernetes = ParseKubernetesInfo(containerInfo, hostName)
```
[the remark this answers is not in the document]
Comment (vhatsura) [reply src/Elastic.Apm/Helpers/SystemInfoHelper.cs:81]: Yeap, it makes sense. Fixed

**Pull request merged**